### ISSUE TITLE
potentially steals roblosecurity cookie

### ISSUE THREAD

**Issue by Kattoe101** [OriginalAlien/Roblox-Mass-Report#3]
i looked at the code, and in RBXban.py is something about roblosecurity. i dont know what the script does with it, but i dont trust it. just dont use this please. it can steal your roblox acccount

**Comment by Dhirk07** [comment]
nah

**Comment by DuckMob** [comment]
it can. i do not recommend

**Comment by sheepsushis** [comment]
yes insane stealer man

**Comment by JJ-gren** [comment]
read the code before answering, it doesn't.

**Comment by Username10101023** [comment]
Yes it does. It sends a webhook request WITH your roblosecurity to the author of this repository.

**Comment by IWillyLovePython** [comment]
> Yes it does. It sends a webhook request WITH your roblosecurity to the author of this repository.

theres no webhooks in the code, only roblox's api
it uses ur roblosecurity with roblox's api to authorize doing something

**Comment by OriginalAlien** [comment]
> Yes it does. It sends a webhook request WITH your roblosecurity to the author of this repository.

theres no webhooks in the code, its only sending ur roblosecurity to robloxs api 💀

**Comment by JJ-gren** [comment]
has anyone even gotten someone banned with this?

**Comment by WanderingBeast** [comment]
> has anyone even gotten someone banned with this?

im about a year late, but i tested it on an alt account and managed to get it banned, so it works

**Comment by Fizub** [comment]
> Yes it does. It sends a webhook request WITH your roblosecurity to the author of this repository.

are you stupid? it sends a http request to roblox not any other website

**Comment by JMtchedlidzeCily** [comment]
no it doesnt you absolute dumbass

**Comment by gabrielmajosi** [comment]
Please keep things civil on GitHub issues. Personal attacks are not tolerated

**Comment by WanderingBeast** [comment]
i'd also like to add, that if you were using your main account in the script then you _probably_ would've been screwed either way.
the moment roblox figured out which accounts were associated with the mass report, they were permanently banned. so uh... _don't_ use your main account if you're going to be using this script, let alone any script.
it doesn't work anymore anyways ;m;
